README.md

```diff
@@ -73,32 +73,33 @@ The following table clarifies the current status of the providers according to t
 
 | Provider | Status | Maintainers | 
 | -------- | ------ | ----------- |
-| Google Cloud DNS | Stable | |
-| AWS Route 53 | Stable | |
+| Akamai FastDNS | Alpha | |
 | AWS Cloud Map | Beta | |
+| AWS Route 53 | Stable | |
 | AzureDNS | Beta | |
 | CloudFlare | Beta | |
-| RcodeZero | Alpha | |
+| CoreDNS | Alpha | |
 | DigitalOcean | Alpha | |
-| Hetzner | Alpha | @21h |
 | DNSimple | Alpha | |
-| Infoblox | Alpha | @saileshgiri |
 | Dyn | Alpha | |
-| OpenStack Designate | Alpha | |
-| PowerDNS | Alpha | |
-| CoreDNS | Alpha | |
 | Exoscale | Alpha | |
-| Oracle Cloud Infrastructure DNS | Alpha | |
+| Google Cloud DNS | Stable | |
+| Hetzner | Alpha | @21h |
+| Infoblox | Alpha | @saileshgiri |
+| Istio | Alpha | @tariq1890|
```

Comment:
I think Istio is an external-dns "source" and this table only contains external-dns "providers". Maybe create a separate table that lists the maintainers for external-dns "sources"? What do you think?

Reply:
Oh yes, good point. I think we should a restructure the document a bit. 

Probably have a separate table for Sources and their respective maintainers?

Reply:
Sounds fine to me, but I'm not one of the maintainers of this project.